### PR TITLE
Update configure-email-forwarding.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the Mail forwarding wordings as it was confusing a little bit.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4667

File: microsoft-365/admin/email/configure-email-forwarding.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ You must be an Exchange administrator or Global administrator in Microsoft 365 t
 
 1. In the admin center, go to the **Users** \> **[Active users](https://go.microsoft.com/fwlink/p/?linkid=834822)** page.
 
-2. Select the name of the user whose email you want to forward to open the properties page.
+2. Select the name of the user whose email you want forwarded, then open the properties page.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/configure-email-forwarding.md]:
```suggestion
2. Select the name of the user whose email you want to forward, then open the properties page.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
@kwekuako Gentle reminder.

**Pull request merged**